LICENSE.txt

```diff
@@ -12,3 +12,27 @@ Unless required by applicable law or agreed to in writing, software distributed
 under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
```

Comment:
@Eilon I just appended the ANCM LICENSE file to IISIntegration's, would you prefer I mention "IISIntegration uses two licenses files..."